crates/oq3_semantics/examples/semdemo.rs

```diff
@@ -141,7 +141,7 @@ fn main() {
 
         Some(Commands::ParseGreen { file_name }) => {
             let (green_node, syntax_errors) = parse_text(&read_example_source(file_name));
-            println!("{:?}", green_node);
+            println!("{green_node:?}");
```

Comment:
This must be a new-ish clippy feature. In any case, it's nice that you changed all of them.